src/agent/commands/results.publish.ts

```diff
@@ -45,7 +48,10 @@ export class ResultsPublishCommand implements cm.IAsyncCommand {
             releaseEnvironmentUri: this.executionContext.variables["release.environmentUri"],
             releaseUri: this.executionContext.variables["release.releaseUri"],
             platform: platform,
-            config: config
+            config: config,
+			runTitle: runTitle,
+			fileNumber: fileNumber,
```

Comment:
We need the file number for attaching to default titles too. Then we would have to extract it out of run title. I think this is fine given our requirements.